### ISSUE TITLE
[Blog] - [Author] - Author's Blog Posts Section

### ISSUE THREAD

**Issue by danielfmiranda** [MozillaFoundation/foundation.mozilla.org#8798]
![](https://user-images.githubusercontent.com/18314510/171497670-507e806c-538f-42ff-8ca1-5efe0221adfe.png)

Using the new card component from #8726, we should render all of the authors blog posts to render like the image above.

If applicable, a load more results button should also render, that will append more blog posts once clicked.

## Dev Steps:
* [ ] Create a new fragment template to house this component
* [ ] Filter and return all blog posts created by this author
* [ ] Render 9 blog posts initially on the front end
* [ ] Add the load more button functionality
* [ ] Once template is complete and working, using the new blog card template, we can then include this in the new author detail page template from #8796 

## Acceptance Criteria:
* [ ] Displays Author's Blog posts using #8726 
* [ ] Cards display and stack as expected on different screen sizes
* [ ] 9 Cards on initial render if possible
* [ ] If applicable, load more results button renders and will append more entries when clicked.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/TP1-285)
